### DISCUSSION TITLE
How to,  by default, skip the prompt to overwrite files

### DISCUSSION THREAD

**Discussion by freyja-ai** [twpayne/chezmoi#4142] [General]
Hi everyone,

Is it possible to prevent chezmoi from asking, by default, if a destination file needs to be overwritten, and just skip this file completely, even if it is in a modified state - except some specific files?

For example, if I run

    chezmoi apply -S $SOURCE -D $DEST

and that creates a file 'test.txt' in the destination, then after I modify test.txt, the same command should complete without prompting to overwrite the file with the source state, unless I specifically "open" this file for potential overwrites. I don't want to use .chezmoiignore in the source - and I don't think I can use .chezmoiignore in the destination.

Does anyone know if this is possible?

For now, my "solution" is to create a wrapper script around chezmoi apply which symlinks a .chezmoiignore file from $DEST to $SOURCE, runs the apply command, then removes the symlink - but that feels incredibly dirty.

Thanks!

**Comment by twpayne** [comment]
Use the [`create_` attribute](https://www.chezmoi.io/reference/source-state-attributes/). See also #4125.

**Reply by freyja-ai** [reply]
Exactly what I need - thanks a bunch! And, thanks for chezmoi - it's fantastic.
